src/frontends/pytorch/src/frontend.cpp

```diff
@@ -218,8 +218,8 @@ void FrontEnd::normalize(const std::shared_ptr<ov::Model>& model) const {
     manager.register_pass<ov::frontend::pytorch::pass::RemovePackingOps>();
     manager.register_pass<ov::pass::RemoveMultiSubGraphOpDanglingParamsResults>();
     manager.register_pass<ov::pass::ReverseShapeAndTypeInfer>();
-    // Second pass of AlignTypesRemoval after all converting transformations
-    manager.register_pass<ov::frontend::pytorch::pass::AlignTypesRemoval>();
+    // Second pass of ConvertConvertPromoteTypes after all converting transformations
+    manager.register_pass<ov::pass::ConvertConvertPromoteTypes>();
```

Comment:
This pass should be moved to regular transformations that are called in plugin, so if user has set types after calling `convert_model` he got a working model.

Reply:
Pass is also in regular common transformations. Though it wouldn't hurt to leave it there just in case, but I guess it shouldn't break anything